taichi/transforms/make_thread_local.cpp

```diff
@@ -0,0 +1,150 @@
+#include "taichi/ir/ir.h"
+#include "taichi/ir/transforms.h"
+#include "taichi/ir/analysis.h"
+#include "taichi/ir/visitors.h"
+
+TLANG_NAMESPACE_BEGIN
+
```

Comment:
nit: anonymous namespace

```suggestion

namespace {

```